video_xblock/static/js/video_xblock.js

```diff
@@ -0,0 +1,41 @@
+/* Javascript for VideoXBlock. */
+function VideoXBlock(runtime, element) {
+
+  var handlerUrl = runtime.handlerUrl(element, 'save_player_state');
+  window.videoXBlockSaveHandlers = window.videoXBlockSaveHandlers || {};
+  window.videoXBlockSaveHandlers[element.attributes['data-usage-id'].value] = handlerUrl;
+  window.videoXBlockListenerRegistered = window.videoXBlockListenerRegistered || false;
+
+  function saveState(handlerUrl, state) {
+    // Saves video player satate by POSTing it to VideoXBlock handler
+    $.ajax({
+      type: "POST",
+      url: handlerUrl,
+      data: JSON.stringify(state),
+    })
+    .done(function() {
+      console.log('Player state saved successfully');
```

Comment:
I thought one does not end a log line with a full stop... But let's make grammar nazis happy. :)